src/daft-functions-tokenize/src/bpe.rs

```diff
@@ -125,7 +125,7 @@ fn get_builtin_bpe(name: &str) -> Option<DaftBPE> {
 }
 
 // This function is templated because tiktoken-rs uses a special hasher for the HashMap.
-fn parse_tokens<H>(s: &str) -> DaftResult<HashMap<Vec<u8>, usize, H>>
+fn parse_tokens<H>(s: &str) -> DaftResult<HashMap<Vec<u8>, u32, H>>
```

Comment:
The old API expected usize for some APIs and u32 for others, which meant we had to do casting. Later versions made it consistent to u32